src/WormholeJoin.sol

```diff
@@ -224,6 +242,34 @@ contract WormholeJoin {
             vat.slip(ilk, address(this), -int256(amtToPayBack));
         }
         debt[sourceDomain] -= int256(batchedDaiToFlush);
+        Paid[sourceDomain] += batchedDaiToFlush;
         emit Settle(sourceDomain, batchedDaiToFlush);
     }
+
+    /**
+    * @dev Rectify bad debt by pulling dai from the vow.
+    * @param sourceDomain domain to rectify
+    * @param era the era in which the bad debt occurred.
+    **/
+    function rectify(bytes32 sourceDomain, uint256 era) external {
```

Comment:
I think we could find ourselves in a situation where `Paid[srcDomain]` is larger or equal to `debtQueue[srcDomain][era]` for all expired `era` s (hence no rectification possible for these eras), yet in aggregate the sum of these debtQueue values could be greater than `Paid` (hence we do have bad debt).

Example with `ttl[srcDomain]` = 8 days:
Day 0: 1m DAI minted
Day 1: 1m DAI minted
Day 7: 1m DAI settled
Day 8: 0 DAI settled. 
So on day 8, `Paid = 1m DAI >= debtQueue[srcDomain][day 0] = 1m DAI` and likewise `Paid >= debtQueue[srcDomain][day 1]`, implying that the rectification can't happen for either day 0 or 1.